dependencies/default/pom.xml

```diff
@@ -71,7 +71,8 @@
     <log4j.version>1.2.17</log4j.version>
     <log4j2.version>2.19.0</log4j2.version>
     <logback.version>1.2.11</logback.version>
-    <mockito.version>4.0.0</mockito.version>
+    <bytebuddy.version>1.12.18</bytebuddy.version>
```

Comment:
we should sort the dependency alphabet order

Reply:
fixed